pkg/node/node.go

```diff
@@ -94,6 +99,11 @@ func (n *Node) setup() {
 	//Make sure we have a config
 	n.Config = &models.Config{}
 	n.Config.MustLoad()
+
+	if n.Config.Version {
+		log.Fatalf("Version is: %s", Version)
```

Comment:
Shouldn't it be a `Printf` instead?

Reply:
No i dont want the application to start when we ask it to print the version